packages/@tinacms/mdx/src/parse/acorn.ts

```diff
@@ -120,28 +120,30 @@ const extractScalar = <
 
 const extractObject = <T extends Extract<TinaField, { type: 'object' }>>(
   attribute: ExpressionStatement,
-  field: T
+  field: T,
+  imageCallback: (image: string) => string
```

Comment:
Should we track this (add a todo?) and make an issue to refactor this at a latter stage?